docs/v2/spec/2018_Public_TS_Data_Dictionary.csv

```diff
@@ -0,0 +1,13 @@
+﻿Field Name,Field Data Type,Valid Values

```

Comment:
For all the rows in valid values, please only list codes (numeric, NA, or Exempt). Descriptions of the data fields are handled elsewhere.

Reply:
Sure - changes will be reflected in the next commit.